### ISSUE TITLE
Legacy tray support disables VRR

### ISSUE THREAD

**Issue by sashaaaaaaaaa** [ubuntu/gnome-shell-extension-appindicator#441]
Basically if I enable legacy tray support VRR stops working. You can test this with vrrtest and if your monitor has a feature to show the current refresh rate. This is on Gnome 44.1 on X11.

**Comment by CubeTheThird** [comment]
I personally am not able to reproduce this. Perhaps it differs depending on the type of GPU drivers? I'm on Nvidia proprietary drivers.

**Comment by Trombach** [comment]
I can confirm this on both X11 and Wayland (both Gnome). If legacy tray icon support is enabled, VRR simply doesn't work. Happy to provide logs and more info

**Comment by Whayme** [comment]
It does not cause any issues for me on X11 but does on Wayland using the experimental patch.  If I install the extension through my package manager and bring in the libappindicator-gtk2 optional dependency, it works just fine without breaking anything.  If I install it from GNOME Extensions, it causes this issue.  You should see if this still occurs with the dependencies.

**Comment by DeedleFake** [comment]
@CubeTheThird Quite probably. I'm on AMD and I _do_ have this problem in Wayland.

**Comment by CubeTheThird** [comment]
So, strangely, despite having now switched to an AMD card, I still do not experience this. HOWEVER, even more strangely, I DO experience it when I turn Legacy Tray support OFF.

**Comment by italoghost** [comment]
This happens to me on both X11 and Wayland (with mutter-vrr patch). Disabling Legacy Tray seemed to work.

**Comment by koplo199** [comment]
The breakage comes from `gnome-shell` itself, I opened an issue at: https://gitlab.gnome.org/GNOME/gnome-shell/-/issues/6955

**Comment by SpoOokY1983** [comment]
YES!!!! I almost deleted my entire system as the Star Citizen Launcher always killed VRR under X11 and Wayland. After having deactivates the damn extension, it works again! So glad I have found this topic!!!
